data/uhc_pack/functions/pre_generation/tick.mcfunction

```diff
@@ -1,3 +1,2 @@
-
-execute if score UHC uhcTick matches 2.. run function uhc_pack:pre_generation/move_player
-execute if score UHC uhcTick matches 2.. run scoreboard players set UHC uhcTick 0
+function uhc_pack:pre_generation/stage/step
```

Comment:
Can `pre_generation/stage/step` be inclined into this function?

Reply:
That should be doable, yep :)